.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature]"
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Which component?**
```

Comment:
I think we should remove this section, and encourage the submitter to point to specific method/components he/she has in mind in the next section (solution) instead.  Feature requests usually come from people who aren't familiar with the code base; if they are given an explicit chance to chime in, it can derail the discussion into a rathole.

Reply:
not necessarily.  I mostly think this is for internal to start to keep track of our TODO.

Reply:
OK then.  Good to go from my side.